### ISSUE TITLE
Get student username list 

### ISSUE THREAD

**Issue by milen-sage** [Sage-Bionetworks/minidream-r-env#17]

**Comment by milen-sage** [comment]
@BrunoGrandePhD could you check if you have access to this team:

https://www.synapse.org/#!Team:3424629

That lists the mini-DREAM student synapse IDs we currently have.

**Comment by BrunoGrandePhD** [comment]
Teams and their membership are always public, so I should be fine with posting usernames here for posterity. 

I cross-referenced the team members with the participants spreadsheet I was sent. I was able to account for all but one participant. I was able to find the remaining participant by using the People Search on Synapse. 

@milen-sage: I think Shannon needs to invited `paulsonl18` to the above team. I think they're the 25th student. 

```
aa71381
aaronzou8
ajperch15
alex_arreola
amonell
autumn
bbxchang
chelseafaithmariano
christinecannon
cortbreuer
emma_wolcott
esha9801
harrise2
jocelyn_hsu
joycelyn.williams
m3arredo
mgrahn
miajeffris
mpiloj
mquan
nih22
paulsonl18
sylvia1
trevor24jones
willthrill
```
